Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/MainActivity.kt

```diff
@@ -58,11 +62,14 @@ class MainActivity : AppCompatActivity(), HasAndroidInjector {
 
     @Inject lateinit var deadmanScheduler: DeadmanNotificationScheduler
     @Inject lateinit var contactDiaryWorkScheduler: ContactDiaryWorkScheduler
+    @Inject lateinit var settings: ContactDiarySettings
```

Comment:
Should this be part of a viewmodel?

Reply:
I will check if possible, just took it as is from the contact diary activity